gateway-service/src/test/java/com/ca/mfaas/gateway/ribbon/GatewayRibbonLoadBalancingHttpClientTest.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+package com.ca.mfaas.gateway.ribbon;
+
+import com.netflix.appinfo.InstanceInfo;
+import com.netflix.client.config.DefaultClientConfigImpl;
+import com.netflix.client.config.IClientConfig;
+import com.netflix.loadbalancer.*;
+import com.netflix.niws.loadbalancer.DiscoveryEnabledServer;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.netflix.ribbon.DefaultServerIntrospector;
+import org.springframework.cloud.netflix.ribbon.ServerIntrospector;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
+
+
+@Import(GatewayRibbonLoadBalancingHttpClientTest.TestConfiguration.class)
+public class GatewayRibbonLoadBalancingHttpClientTest {
```

Comment:
Please, throw exception instead of wrapping in the tests.

Reply:
For this class, there is no need to setup spring env. May I ask to redesign this class like this?
https://github.com/zowe/api-layer/blob/48bfe42dac0117c7a3a237d17abad8cf219e5106/gateway-service/src/test/java/com/ca/mfaas/gateway/ribbon/GatewayRibbonLoadBalancingHttpClientTest.java